services/provider/server/server.go

```diff
@@ -180,11 +180,11 @@ func (s *OCSProviderServer) GetStorageConfig(ctx context.Context, req *pb.Storag
 			return nil, status.Errorf(codes.Internal, "failed to get external resources. %v", err)
 		}
 
-		channelName, err := s.getOCSSubscriptionChannel(ctx)
+		subscriptionSpec, err := s.getOCSSubscriptionSpec(ctx)
```

Comment:
We should not create an entire spec to get the channel name. preferably have a func to get the channel name then use it here and when creating the spec.